samples/dashboards/demo/aws-management/demo.details

```diff
@@ -0,0 +1,12 @@
+---
+name: AWS Management
```

Comment:
I'm wondering if we even should use the `aws` keyword here 🤔 
Maybe all should be more general like `Server monitoring dashboard`

Reply:
On the one hand, it makes sense, however on the other hand it's the real use case.
@oysteinmoseng what do you think?